src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -3588,6 +3613,11 @@ subroutine wrfv3_netcdf(fv3filenamegin)
          call gsi_fv3ncdf_write(grd_fv3lam_phyvar_ionouv,gsibundle_fv3lam_phyvar_nouv,&
                                 add_saved,fv3filenamegin%phyvars,fv3filenamegin)
       end if
+      if( if_model_fed ) then
```

Comment:
Again,  please consider the case when both if_model_dbz and if_model_fed exist .  only one write is needed. 
A test of if grd_fv3lam_phyvar_ionouv is allocated may work.

Reply:
@TingLei-NOAA 
I will change to one call that only update gsibundle_fv3lam_phyvar_nouv once: 
      if( if_model_dbz .or. if_model_fed ) then
         call gsi_fv3ncdf_write(grd_fv3lam_phyvar_ionouv,gsibundle_fv3lam_phyvar_nouv,&
                                add_saved,fv3filenamegin%phyvars,fv3filenamegin)
      end if